pkgs/by-name/ge/gentium-book/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
Could you use `finalAttrs:` pattern please?

Reply:
Fixed